app/login/page.tsx

```diff
@@ -230,8 +257,8 @@ const Login = () => {
                     textAlign: "center",
                     marginTop: "0px",
                     color: "rgba(17, 20, 45, 1)",
-                    
-                  }}>
+                  }}
```

Comment:
There is some unnecessay code has been committed here. Please check and commit only the necessary changes. Please don't delete and readd codes that do not need to changes.